### PR TITLE
CSRF token as singleton

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ✔️
| Fixed issues  | -

Static CSRF Token class replaced to singleton class.

### CODE REVIEW

**Review by romkatsu — APPROVED** [review]

File: src/TokenMask/CsrfTokenMask.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Csrf\TokenMask;
+
+use Yiisoft\Security\TokenMask;
+
+final class CsrfTokenMask implements CsrfTokenMaskInterface
```
Comment (samdark) [inline src/TokenMask/CsrfTokenMask.php:9]: Why do we need this one? There's a correct way to do masking security-wise and I see no reason not to use it directly.

Comment (yiiliveext) [reply src/TokenMask/CsrfTokenMask.php:9]: For example, you want to add a time interval prefix to the token. This is a way to do this. Another way is `CsrfTokenInterface`

Comment (samdark) [reply src/TokenMask/CsrfTokenMask.php:9]: That isn't a good idea. In this case we should not give any ability to customize masking at all.

Comment (samdark) [reply src/TokenMask/CsrfTokenMask.php:9]: Let's remove the interface.

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**